resources/assets/sass/app.scss

```diff
@@ -433,3 +433,141 @@ legend {
     from { transform: scale(1) rotate(0deg);}
     to { transform: scale(1) rotate(360deg);}
 }
+
+/* GitHub Login button styles
+ * Using: https://github.com/necolas/css3-social-signin-buttons
+ */
+.btn-auth,
+.btn-auth:visited {
+    background: #fff;
+    border: 1px solid #999;
+    border-radius: 20px;
+    box-sizing: content-box;
+    color: #222;
+    cursor: pointer;
+    display: inline-block;
+    font-size: 14px;
+    height: 22px;
+    line-height: 22px;
+    margin: 0;
+    padding: 0 1em;
+    position: relative;
+    text-align: center;
+    text-decoration: none;
+    user-select: none;
+    white-space: nowrap;
+    -webkit-box-sizing: content-box;
+    -moz-box-sizing: content-box;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    /* iOS */
+    -webkit-appearance: none; /* 1 */
+    /* IE6/7 hacks */
+    *overflow: visible;  /* 2 */
+    *display: inline; /* 3 */
+    *zoom: 1; /* 3 */
+}
+
+.btn-auth:hover,
+.btn-auth:focus,
+.btn-auth:active {
+    color: #222;
+    text-decoration: none;
+}
+
+.btn-auth:before {
+    background: url(https://raw.githubusercontent.com/necolas/css3-social-signin-buttons/gh-pages/auth-icons.png) no-repeat 99px 99px;
```

Comment:
Let's move this background image to the repo instead of loading it from a third party URL.